getDataTableReader.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Get Data (TableDataReader)</title>
+
+    <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
+
+    <script type="module">
+      // TableauEventType represents the type of Tableau embedding event that can be listened for.
+    //  import { TableauEventType } from "https://public.tableau.com/javascripts/api/tableau.embedding.3.latest.js";
+    // TODO: change import with 22.4 release on Public
+      import { TableauEventType } from 'https://devplat.tableautest.com/javascripts/api/tableau.embedding.3.latest.js'
+
+      async function handleFirstInteractive (e) {
+        const getDataButton = document.getElementById('get-data');
+
+        let pageSelector = $('#pageSelect');
+        pageSelector.prop('disabled', false);
+
+        let pageNumber = '${pageSelector.val()}';
+
+        //  update the page data upon a selector change.
+        pageSelector.change(function () {
+          getDataAsync(pageSelector.val())
+        });
+
+        // default values if null.
+        if (pageSelector.val() === null) {
+          pageNumber = 0
+        };
+
+        // Get the first page of data
+        getDataButton.onclick = async () => {
+          getDataAsync(pageNumber)
+        }
+      }
+
+      // Get the underlying data using the DataTableReader
+      async function getDataAsync (pageNumber) {
+        // Get data from a specified sheet
+        const sheet = viz.workbook.activeSheet.worksheets.find(
+          sheet => sheet.name === 'Storm Map Sheet'
+        );
+        const datasource = await sheet.getDataSourcesAsync();
+
+        // Get the logical tables from the first data source
+        const logicalTables = await datasource[0].getLogicalTablesAsync();
+
+        // Loop through each table that was used in creating this data source
+        logicalTables.forEach(function (table) {
+          console.log(`Name of logical table: ${table.caption}`);
+          document.getElementById(
+            'info'
+          ).innerHTML = `<p>Name of logical table: ${table.caption}</p>`
+        });
+
+        // Get the data from the first logical table
+        const logicalTableData = await datasource[0].getLogicalTableDataAsync(
+          logicalTables[0].id
+        );
+        console.log(logicalTableData);
+        console.log(`Total row count: ${logicalTableData.totalRowCount}`);
```

Comment:
Do we want to have a comment here that it will be a maximum of 10,000 rows?

Reply:
Just added the 10,000 rows comment in the right place (right before we display `totalRowCount` 
`// There is a maximum of 10,000 rows`

Reply:
How about
// There is a maximum of 10,000 rows for getLogicalTableDataAsync